src/app/zap-templates/common/override.js

```diff
@@ -76,9 +76,12 @@ function atomicType(arg)
     return 'chip::Percent100ths';
   case 'epoch_us':
   case 'systime_us':
+  case 'posix_ms':
+  case 'systime_ms':
```

Comment:
Please also update

```
scripts/py_matter_idl/matter_idl/generators/types.py
```

with the new types. It looks like a copy&paste works there as well.

Reply:
@andy31415 Done.